blockchain/package.json

```diff
@@ -14,7 +14,9 @@
   "license": "ISC",
   "description": "",
   "devDependencies": {
+    "@nomicfoundation/hardhat-ignition-ethers": "^0.15.7",
```

Comment:
@Emmanuel-Darko Does this package really need to be installed?

Reply:
Yeah @mikehammond
I followed the deployment guide on hardhat

![alt text](image.png)